recipe/meta.yaml

```diff
@@ -35,7 +35,7 @@ requirements:
     - clang_osx-64  # [osx]
     - m2w64-toolchain  # [win]
     - matplotlib >=2.0.2
-    - netcdf4 >=1.1.9
+    - netcdf4 >=1.1.9,<=1.4.1
     - numpy >=1.9.1
```

Comment:
We should follow https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L462 and pin numpy.

Reply:
Sure, happy to pin numpy. Is that simply a matter of changing 
```
    - numpy >=1.9.1
```
to
```
    numpy:
      - 1.9    # [linux64 or osx]
      - 1.11   # [aarch64 or ppc64le]
      - 1.11   # [win]
```
?

Reply:
I think it's
```yaml
    - numpy >=1.9  # [linux64 or osx]
    - numpy >=1.11  # [win]
```

Not sure if we need the others.